apis/compute/v1alpha1/computeregiontargettcpproxy_types.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	refs "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/apis/k8s/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var ComputeRegionTargetTCPProxyGVK = GroupVersion.WithKind("ComputeRegionTargetTCPProxy")
```

Comment:
Just to confirm, do you plan to add the global  targetTCPProxy as a different kind?

Reply:
global is already a [TF-based resource](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/crds/compute_v1beta1_computetargettcpproxy.yaml#L25), I want to take this chance to support regional and split it from global.
We also have some compute regional resources(alpha): https://github.com/search?q=repo%3AGoogleCloudPlatform%2Fk8s-config-connector%20compute_v1alpha1_computeregion&type=code